web/src/package.json

```
@@ -89,6 +89,7 @@
     "moment": "^2.22.2",
     "react": "16.8.6",
     "react-apollo": "2.5.6",
+    "react-apollo-hooks": "^0.4.5",
```

I think we were going to lock versions in `package.json`
```suggestion
    "react-apollo-hooks": "0.4.5",
```